src/storm-pars/modelchecker/region/SparseParameterLiftingModelChecker.cpp

```diff
@@ -313,15 +313,29 @@ namespace storm {
                     std::vector<storm::storage::ParameterRegion<typename SparseModelType::ValueType>> newRegions;
 
                     // Check whether this region needs further investigation based on the bound of the parent region
-                    bool investigateBounds = !currBound || (minimize && currBound.get() < value.get() - storm::utility::convertNumber<ConstantType>(precision))
-                                             || (!minimize && currBound.get() > value.get() + storm::utility::convertNumber<ConstantType>(precision));
+                    bool investigateBounds = !currBound;
+
+                    if (!investigateBounds && absolutePrecision) {
+                        investigateBounds = (minimize && currBound.get() < value.get() - storm::utility::convertNumber<ConstantType>(precision))
+                                            || (!minimize && currBound.get() > value.get() + storm::utility::convertNumber<ConstantType>(precision));
+                    } else if (!investigateBounds && !absolutePrecision){
+                        investigateBounds = (minimize && (currBound.get() * (1 + storm::utility::convertNumber<ConstantType>(precision)) < value.get()))
+                                            || (!minimize && (currBound.get() * (1 + storm::utility::convertNumber<ConstantType>(precision)) > value.get()));
```

Comment:
I think it has to be a minus in the maximizing case:
```suggestion
                                            || (!minimize && (currBound.get() * (1 - storm::utility::convertNumber<ConstantType>(precision)) > value.get()));
```